bd-core-prd/ingress.yaml

```diff
@@ -18,6 +18,13 @@ spec:
     - host: api.prd.osrsbotdetector.com
       http:
         paths:
+          - path: /v2 # this supports migration to /v2 api endpoints
+            pathType: Prefix
+            backend:
+              service:
+                name: public-api-svc
```

Comment:
I almost have to wonder if it wouldn't just be simpler to **not** add a second ingress and to instead to point this `path: /v2` at the api-v2 service.  In this design those pods will be required to run from the same `bd-prd` namespace as the ingress object, but as mentioned in another comment it seems to already be a bit ambiguous whether this service is prod or dev.

Reply:
the second path is only because the /docs is not available if we do the redirect.
the redirect is temporary and in the future we point everything to the public api